tests/storage/restricted_namespace_cloning/conftest.py

```diff
@@ -252,3 +287,19 @@ def vm_for_restricted_namespace_cloning_test(
     ) as vm:
         running_vm(vm=vm, wait_for_interfaces=False)
         yield vm
+
+
+@pytest.fixture()
+def user_has_get_permissions_in_source_namespace(
+    namespace, unprivileged_client, data_volume_multi_storage_scope_module
+):
+    assert list(
+        DataVolume.get(
+            namespace=namespace.name, name=data_volume_multi_storage_scope_module.name, dyn_client=unprivileged_client
+        )
+    )[0]
+
+
+@pytest.fixture()
+def verify_image_permissions(request):
```

Comment:
no verbs in fixture name please. Also please rename this fixture. You are reading a param not verifying anything.